app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,65 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px; }
+
+body, p, ol, ul, td {
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px; }
+
+pre {
+  background-color: #eee;
+  padding: 10px;
+  font-size: 11px; }
+
+a {
+  color: #000; }
+
+a:visited {
+  color: #666; }
+
+a:hover {
+  color: #fff;
+  background-color: #000; }
+
+th {
+  padding-bottom: 5px; }
+
+td {
+  padding: 0 5px 7px; }
+
+div.field,
+div.actions {
+  margin-bottom: 10px; }
+
```

Comment:
Everything above line 35 seems odd to me, presumably these are Rails defaults. But we want Bootstrap defaults! I assume Bootstrap also defines sensible defaults for these elements.

Reply:
I looked at last year's project and there were the same values. It could be that we used rails defaults back then too. But it did not seem to bother anyone then. So maybe it's just fine?

Reply:
The problem with the application not looking like bootstrap was that the application stylesheet did not import bootstrap. I fixed that, so these defaults should be fine, as I mentioned above